gcc/doc/install.texi

```diff
@@ -312,6 +312,12 @@ documentation including the target @code{SYSTEM} definition module.
 If Python3 is unavailable Modula-2 documentation will include a target
 independent version of the SYSTEM modules.
 
+@item @anchor{gccrs-prerequisite}gccrs
+
+The official Rust compiler is required for building various parts of
```

Comment:
We should mention that cargo is required alongside rustc.

Reply:
Thanks!